froide_fax/utils.py

```diff
@@ -52,36 +50,27 @@ def get_signature(user):
     return signature
 
 
-FAX_MEDIA_SALT = 'fax_media_url'
-FAX_CALLBACK_SALT = 'fax_callback_url'
+FAX_MEDIA_SALT = "fax_media_url"
+FAX_CALLBACK_SALT = "fax_callback_url"
 
 
 def get_media_url(att):
-    # Telnyx doesn't seem to follow redirects
-    # Give short-lived direct authenticated URL
-    # instead of permanent signed URL that redirects
-    return att.get_absolute_domain_file_url(authorized=True)
```

Comment:
This change here seems to have been lost, but it's essential for Telnyx integration to work.

Reply:
Oh. I thought I started with a fresh pull on the master-branch. I'll look into that.

Reply:
I cherry-picked that one back in.